src/renderer/components/avatar/avatar.tsx

```diff
@@ -72,9 +72,21 @@ function getIconString(title: string) {
 export function Avatar(props: Props) {
   const { title, width = 32, height = 32, colorHash, children, background, ...settings } = props;
 
+  const getBackgroundColor = () => {
```

Comment:
Why not just make this an immediately called function since the result never changes?

Reply:
Good point! Will change that in `HotbarIcon` -> `EntityIcon` refactoring.